### ISSUE TITLE
[1.15.2]

### ISSUE THREAD

**Issue by TheUntamed** [An-Sar/PrimalCore#374]
Are there any plans in porting this mod to MC 1.15.2?

**Comment by An-Sar** [comment]
yes, however based on current development time a stable 1.16 release is the ideal target

**Comment by TheUntamed** [comment]
MC 1.15.2 is not stable?

**Comment by An-Sar** [comment]
it's not about stability but just a function of time and volume of work, i said stable 1.16 meaning anything before that would be under heavy development.

**Comment by TheUntamed** [comment]
I can definitely understand that.
I'm trying to develop a modpack and the time it takes is insane.
That's also why your message made me shiver. After 1.12 it's really hard to develop something.
1.13 was skipped like expected. Then People including me started with 1.14 but that version turned out to be a bug feast.
So 1.15 looked like it could be the next 1.12.
To hear people will skip that too leaves me in doubt that we will ever have a new 1.12 (or better) in modded Minecraft.
Not even talking about the split because of Forge/Fabric.

**Comment by An-Sar** [comment]
i hear that. i am also interested in targeting 1.16 due to the new nether content so it makes a natural objective. 

i got about 60% updated to 1.14 when it became clear 1.15 was on the way...

**Comment by TheUntamed** [comment]
Mojang should stop updating :o

**Comment by lamoyrl90** [comment]
I just hope Mojang doesn't release a new version that makes Forge redo its entire codebase.

Good to know you're still working on it, though! I was getting worried looking at the mid-2019 last-update status on Curse.

**Comment by Unhook** [comment]
Any update on this?
